### DISCUSSION TITLE
keybindings generated from other plugins are not registered

### DISCUSSION THREAD

**Discussion by jeusdi** [mrjones2014/legendary.nvim#260] [Q&A] [answered]
I'm not aware if next behavior is right.
I've just installed a plugin `echasnovski/mini.surround`. This plugins provide some keybindings in order to deal with surrounding issues. For example, `saiw"`, `sr"'`... you can find command examples on [documentation](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-surround.md#features)...

For example, in order to replace `"` by `'`, you need to type `sr"'`, if you want to replace `[` by `(`, you need to type `sr[(`...

I've took a look on `:Legendary` but above actions are not listed.
Is it right?

**Comment by mrjones2014** [comment, marked as answer]
Yeah, there’s no way to just automagically have it registered in legendary.nvim. You’ll have to add those manually.

There is #183 but I have no concrete implementation plans at the moment.
